types/google-apps-script/google-apps-script.spreadsheet.d.ts

```diff
@@ -1710,8 +1747,8 @@ declare namespace GoogleAppsScript {
             getDisplayValue(): string;
             getDisplayValues(): string[][];
             getFilter(): Filter | null;
-            getFontColor(): string;
-            getFontColors(): string[][];
+            getFontColorObject(): Color;
+            getFontColorObjects(): Color[][];
```

Comment:
This is a new change - note that there is no `getFontColors(s)` in the API, which makes sense since the corresponding setter is `setFontColorObject(s)`

https://developers.google.com/apps-script/reference/spreadsheet/range#getfontcolorobject

Reply:
Oops, getFontColor and getFontColors are **deprecated** but not deleted.

See https://developers.google.com/apps-script/reference/spreadsheet/range#deprecated-methods_1

Please set `@deprecated` to deprecated methods and avoid tsc error for users which use depreacted methods.

```ts
/** @deprecated DO NOT USE */ getFontColor(): string;
/** @deprecated DO NOT USE */ getFontColors(): string[][];
getFontColorObject(): Color;
getFontColorObjects(): Color[][];
```

Reply:
Oh, I actually checked and didn't spot it... oh well, here it is again, sorry for the delay